extensions/adobe/experience/journeyOrchestration/stepEvents/journeyStepEventSupplementalFieldsMixin.schema.json

```diff
@@ -0,0 +1,42 @@
+{
+  "meta:license": [
+    "Copyright 2020 Adobe Systems Incorporated. All rights reserved.",
+    "This work is licensed under a Creative Commons Attribution 4.0 International (CC BY 4.0) license",
+    "you may not use this file except in compliance with the License. You may obtain a copy",
+    "of the License at https://creativecommons.org/licenses/by/4.0/"
+  ],
+  "$id": "https://ns.adobe.com/experience/journeyOrchestration/stepEvents/journeyStepEventSupplementalFieldsMixin",
+  "$schema": "http://json-schema.org/draft-06/schema#",
+  "title": "Journey Step Event Supplemental Fields Mixin",
+  "type": "object",
+  "meta:extensible": true,
+  "description": "Journey Step Event Supplemental Fields Mixin.",
+  "definitions": {
+    "journeyStepEventSupplementalFields": {
+      "properties": {
+        "https://ns.adobe.com/experience/journeyOrchestration/stepEvents": {
+          "title": "Journey Step Event Supplemental Fields",
+          "type": "object",
+          "properties": {
+            "https://ns.adobe.com/experience/journeyOrchestration/supplementalID": {
+              "title": "Supplemental ID",
+              "type": "string",
+              "description": "Supplemental identifier used for multiple re-entrance scenarios"
+            },
+            "https://ns.adobe.com/experience/journeyOrchestration/supplementalNamespace": {
+              "title": "Supplemental Namespace",
+              "type": "string",
+              "description": "Namespace for the supplemental identifier"
+            }
+          }
+        }
+      }
+    }
+  },
+  "allOf": [
+    {
+      "$ref": "#/definitions/journeyStepEventSupplementalFields"
+    }
+  ],
+  "meta:status": "stable"
```

Comment:
Since this is a very new field group, do you think this will evolve? If so it is better to mark it "experimental" so that it is not exposed to all customers.